scripts/scripts.js

```diff
@@ -213,16 +214,27 @@ function buildArticleSocialShare(main) {
   main.append(div(buildBlock('social-share', { elems: [] })));
 }
 
+function hasInlinedSidebar(main) {
+  const sectionMetas = [...main.querySelectorAll('div.section-metadata')];
```

Comment:
I assume we go in reverse order because we expect that `sidebar` is at the end.

Reply:
Yes, that was my thinking that the sidebar, by our modeling, is normally one of the last sections, so the main content comes first. I didn't want to risk looking only at the last section.